### DISCUSSION TITLE
Is this repo abandoned?

### DISCUSSION THREAD

**Discussion by suhaylmv** [motiondivision/motionone#225] [Support]
I'm seeing a lot of issues which remained open for two years. I was thinking of opening an issue because I think I've encountered a bug, but idk if it would make any difference due to this reason

**Comment by rolanday** [comment]
Funny, came here to ask precisely the same question and found this a top of queue. Feels like we're asking a rhetorical question, but would be great to get confirmation one way or  the other. If no answer then I think we've got our answer.

**Comment by Acccent** [comment]
I don't think it's abandoned (@mattgperry can confirm, but he pushed a commit literally the day after this thread was started and that alleviated my own doubts tbh), it's just that I assume the bandwidth to deal with issues and PRs is limited.

**Comment by suhaylmv** [comment]
I'm glad to hear that. I really appreciate all the work Matt has put on motion one and framer motion. However I think he should care more about the image / impressions he is giving to the public. Having so many old open issues (some of them are a year old and don't even have a response) and some things not working on the website (like motion one solid examples, since a very long time), make the repo seem abandoned and push away new people thinking about trying out motion one. If Matt just fixes these two issues (even if he doesn't make commits for several months), I think it would be a big step forward

**Reply by rolanday** [reply]
+1. I'm a framer user on React so mad props to Matt -- it's a fantastic lib. I have no complaints regarding motion -- it's FOSS and the contrib is super appreciated, and prioritizing time spent on framer is the right call IMO. I only take deps on software that have strong active maintenance signals,  so for Solid, I've decided to use gsap and I'm OK to pay license fee. Will prob switch my React project over to it too.

**Comment by John0x** [comment]
As far as I'm aware @mattgperry joined the framer-motion team and therefore has new priorities. I'd expect this project to be abandoned pretty soon in favour of framer-motion. 
Would be great to get an official update on the situation by @mattgperry :)

**Comment by mattgperry** [comment]
This was started long after Framer Motion and many of its innovations have since been folded into it. Likewise subsequent bugfixes have found their way back here. The project isn't going to be abandoned (recent updates prove as much) but bandwidth is low. Likewise Framer Motion has many open bugs, some for years, but it's in active development and as one developer working on both libraries bandwidth is limited. The thing that someone thinks is an urgent issue might often not be something I agree is urgent, aligns with business priorities, or is something interested in solving.

My only caveat to all this is that I will be looking into spinning out the library integrations for this library, doing these was an overstep on my part and I'm more interested in solving common user complaints about WAAPI than I am making essentially super stripped-down versions of Framer Motion for other view libraries.

**Reply by Acccent** [reply]
Ty for the comment! As far as I'm concerned, no problem whatsoever with this approach. And for what it's worth, personally I'm using Svelte and find it easier to just integrate the base library rather than a component wrapper.

I've used GSAP for a long time and have a ton of respect for it, but I really appreciate that Motion One exists and love working with it currently, both because of its syntax and its modernity. And if the plan is to continue to maintain it as a 'DX layer' over WAAPI, along with a few features like springs and scroll, then imo that's all we could ask for.

edit: I would suggest pinning an announcement to make all of this clear to newcomers :)

**Reply by John0x** [reply]
@mattgperry Thank you for the clarification.Totally understandable.
I'm wondering though whether or not the devtools will make their way into framer-motion? I'm actively using framer-motion in a lot of my projects and having some visual editor (especially for animation timings), would be extremely beneficial.

**Comment by rolanday** [comment]
@mattgperry ty again for your amazing work and contribution. I think we can all appreciate needing to prioritize, and I think framer is the right call. It means motion on solid is not an option for me, but that's OK, and frankly the is the price one needs to get comfortable with for moving off something like React. There is no free lunch in life. Thanks for taking the time to chime in.
